rotkehlchen/api/v1/resources.py

```diff
@@ -282,6 +283,17 @@ def _combine_parser_data(
     return data_1
 
 
+def _maybe_get_temp_file(filepath: FileStorage | Path) -> Path:
+    """Checks if filepath is a FileStorage and if so,
+    creates a temporary copied file and returns its path."""
+    if isinstance(filepath, FileStorage):
+        _, tmpfilepath = tempfile.mkstemp()
+        filepath.save(tmpfilepath)
+        return Path(tmpfilepath)
+
+    return filepath
```

Comment:
Shouldn't this happen in `AsyncFilePathSchema`. This function can be a staticmethod in that class